admin_manual/configuration_files/encryption_configuration.rst

```diff
@@ -172,24 +178,4 @@ user's password on the back-end, the user will be prompted to change their ownCl
 to match on their next ownCloud login. The user will need both their old and new passwords 
 to do this. If you have enabled the Recovery Key then you can change a user's password in 
 the ownCloud Users panel to match their back-end password, and then, of course, notify the 
-user and give them their new password.
-
-"Missing requirements" Message on Windows Servers
```

Comment:
Maybe just uncomment it instead of deleting it? Even if windows support was dropped its very likely that this documented issue on windows will get lost if there will be windows support again in the future.